README.md

```diff
@@ -84,7 +84,8 @@ If you have a custom setup use the following options to configure Jest Runner:
 | jestrunner.disableCodeLens | Disable CodeLens feature |
 | jestrunner.codeLensSelector | CodeLens will be shown on files matching this pattern (default **/*.{test,spec}.{js,jsx,ts,tsx}) |
 | jestrunner.codeLens | Choose which CodeLens to enable, default to `["run", "debug"]` |
-| jestrunner.enableYarnPnpSupport Enable if you are using Yarn 2 with Plug'n'Play |  
+| jestrunner.enableYarnPnpSupport Enable if you are using Yarn 2 with Plug'n'Play |
+| jestrunner.exactTestName | Strictly match test name, defaults to `false`. By default multiple tests will be run when test names overlap |
 | jestrunner.yarnPnpCommand | Command for debugging with Plug'n'Play defaults to yarn-*.*js |
```

Comment:
I now realized that I put this in between the `yarnPnp` commands. I was only looking alphabetically. I will adjust and re-push.